pandas/tests/reshape/test_tile.py

```diff
@@ -326,6 +326,12 @@ def test_series_retbins(self):
                                                                 ordered=True)
         tm.assert_series_equal(result, expected)
 
+    def test_qcut_labels_true(self):
```

Comment:
cycle thru a few more things that are not valid (you can use parametrize) , e.g. 'foo', 1